util/gcs/gcs.go

```diff
@@ -172,6 +172,9 @@ const (
 	DefaultACL = false
 	// PublicRead ACL for this upload.
 	PublicRead = true
+	// May cache, but only after verifying.
+	// See https://cloud.google.com/storage/docs/metadata#cache-control
+	ValidateCache = "no-cache"
```

Comment:
Not a huge deal, but I continue to think `NoCache` is a better name for using the `no-cache` value in this header.

What is the motivation for creating an abstraction layer here? It just seems to distance people from understanding what is actually happening.

AKA I think it is better for people to think concretely:
* "I want to upload this file to GCS with a `Cache-Control: no-cache` header" -- and understand how GCS will work with this

Than to have some abstract idea:
* "I want this file to be validated before using the cached version" or something and hope that this abstract understanding matches what GCS will do.

The reality is that the only way for them to be sure about this is to understand a) that `ValidateCache` means `no-cache` and b) how GCS behaves with `no-cache`, at which point this abstraction becomes useless.

Also, using validate seems to conflate with the way this header can use `revalidate`

Reply:
Abstraction isn't what I'm interested in here, and I don't want to abstract in a way that makes it less obvious what this does. (Not being able to mistype "nocache" is nice, but not my goal)
 
What I'm want is for the argument to either describe what it does ("Validate caches before serving them") or, barring that, be strange enough that I'll hit F12, see the documentation link, and remember that "no-cache" is complicated when it doesn't appear complicated.
 
I still think that "no-cache" is not an ideal name for that _header_, although I'd have to take that up with the [RFC](https://datatracker.ietf.org/doc/html/rfc7234#section-5.2).